samza-rest/src/main/java/org/apache/samza/rest/proxy/job/SimpleYarnJobProxy.java

```diff
@@ -40,8 +40,8 @@
   private static final String START_SCRIPT_NAME = "run-job.sh";
```

Comment:
Does this need to use `run-app.sh`? It looks like you changed everything else to `run-app.sh`. I'm not sure what this is used for, but would this also need testing?

Reply:
Updated to run-app.sh as well. 

Tested with `curl -X PUT 'localhost:9139/v1/jobs/wikipedia-feed/1?status=start'`